### PR TITLE
PR for a working WPCAS phase-1.

### LINKED ISSUE #75: PR review for phase 1
Please review the PR: https://github.com/COMPASS-DPG/wpcas-service/pull/74

### PR DESCRIPTION
**Pull request by VamshiBatta07**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -12,88 +12,153 @@ datasource db {
...
+//   id           Int            @default(autoincrement())
+//   departmentId Int            @unique
+//   name         String
+//   description  String?
+//   UserMetadata UserMetadata[]
+//   SurveyConfig SurveyConfig?
+
+//   @@id([id, departmentId])
+//   @@map("admin_department")
+// }
```
Comment (kh4l1d64) [inline prisma/schema.prisma:69]: Remove comments if not necessary.

File: prisma/schema.prisma [hunk 2]
```diff
@@ -12,88 +12,153 @@ datasource db {
...
+  onboardingTime     Int           @default(3)
+  onboardingTimeUnit TimeUnitsEnum @default(MONTH)
+  startTime          DateTime
+  endTime            DateTime
+  isActive           Boolean       @default(false)
+  SurveyForm         SurveyForm[]
+  UserMapping        UserMapping[]
+  // departmentId          Int                    @unique
+  // AdminDepartment       AdminDepartment        @relation(fields: [departmentId], references: [departmentId])
+  // SurveyCycleParameters SurveyCycleParameter[]
```
Comment (kh4l1d64) [inline prisma/schema.prisma:83]: Remove unnecessary comments

File: prisma/schema.prisma [hunk 3]
```diff
@@ -12,88 +12,153 @@ datasource db {
...
+//   endTime        DateTime
+//   surveyForm     SurveyForm[]
+//   isActive       Boolean
+//   surveyConfigId Int?
+//   SurveyConfig   SurveyConfig? @relation(fields: [surveyConfigId], references: [id])
+//   createdAt      DateTime      @default(now())
+//   updatedAt      DateTime      @updatedAt
+
+//   @@map("survey_cycle_parameters")
+// }
```
Comment (kh4l1d64) [inline prisma/schema.prisma:108]: Same as above

File: prisma/schema.prisma [hunk 4]
```diff
@@ -12,88 +12,153 @@ datasource db {
...
-  isNewEmployee Boolean
-  createdAt     DateTime @default(now())
-  updatedAt     DateTime @updatedAt
+  id                         Int               @unique() @default(autoincrement())
+  userId                     String            @unique() @db.Uuid
+  userName                   String            @unique()
+  profilePicture             String?
+  isNewEmployee              Boolean
+  designation                String?
+  dateOfJoining              DateTime
```
Comment (SanchitEsMagico) [inline prisma/schema.prisma:21]: is dateOfJoining same as createdAt?

**Review by kh4l1d64 — APPROVED** [review]
Looks good. Can remove the department related code if not needed, instead of commenting it out.

File: prisma/schema.prisma (hunk 4 above)
Comment (VamshiBatta07) [reply prisma/schema.prisma:21]: No, its the date at which an employee joined in the organization.

File: prisma/schema.prisma (hunk 1 above)
Comment (VamshiBatta07) [reply prisma/schema.prisma:69]: We do not know if this table will be used again. So I just commented it out.

File: prisma/schema.prisma (hunk 2 above)
Comment (VamshiBatta07) [reply prisma/schema.prisma:83]: Keeping them for future reference.

File: prisma/schema.prisma (hunk 3 above)
Comment (VamshiBatta07) [reply prisma/schema.prisma:108]: Keeping them for future reference.

**Review by Ajay-Maury — APPROVED** [review]

**Review by SanchitEsMagico — APPROVED** [review]

**Pull request merged**